Sources/ArcGISToolkit/Components/Form/FormInputs/SwitchInput.swift

```diff
@@ -33,32 +36,23 @@ struct SwitchInput: View {
     @State private var requiredValueMissing = false
     
     /// A Boolean value indicating whether the switch is toggled on or off.
-    @State private var switchState: Bool = false
+    @State private var isOn: Bool = false
```

Comment:
`isOn` seems like a clearer name to me and matches the name of the Toggle's parameter.